Assets/Scripts/UnityServices/Lobbies/LobbyServiceFacade.cs

```diff
@@ -212,8 +212,14 @@ public async Task<(bool Success, Lobby Lobby)> TryQuickJoinLobbyAsync()
         void ResetLobby()
         {
             CurrentUnityLobby = null;
-            m_LocalUser.ResetState();
-            m_LocalLobby?.Reset(m_LocalUser);
+            if (m_LocalUser != null)
+            {
+                m_LocalUser.ResetState();
+            }
+            if (m_LocalLobby != null)
+            {
+                m_LocalLobby.Reset(m_LocalUser);
+            }

```

Comment:
What if m_LocalUser is null? Should this be nested inside the other check?

Reply:
see reply to above comment